packages/widget-message-meet/src/actions/activity.js

```diff
@@ -60,18 +88,19 @@ export function updateActivityText(text) {
   };
 }
 
+
 /**
  * Adds file to message, creates Share activity if not present, starts upload
  *
  * @param {object} conversation - from store
- * @param {Map} activity - from store
+ * @param {Map} activityStore - from store
  * @param {array} files
  * @param {object} spark - spark instance
  * @returns {function}
  */
-export function addFiles(conversation, activity, files, spark) {
+export function addFiles(conversation, activityStore, files, spark) {
   return (dispatch) => {
-    let shareActivity = activity.get(`shareActivity`);
+    let shareActivity = activityStore.getIn([`shareActivity`]);
```

Comment:
Yea, I think I went through a few iterations of the store and didn't get this changed back.